tools/release/github/changelog.py

```diff
@@ -35,6 +35,7 @@ class Changelog:
     key_labels = "labels"
     key_name = "name"
 
+    label_dsip = "dsip"
```

Comment:
could you also change priority in docstring in L28, due to we add new section for our release

Reply:
@SbloodyS seems you missed docstring change

Reply:
Done.